packages/hoppscotch-backend/src/types/InfraConfig.ts

```diff
@@ -24,12 +31,19 @@ export enum InfraConfigEnumForClient {
 
```

Comment:
Can we introduce the exclude const mechanism we use in SHE here as well @mirarifhasan and remove the `InfraConfigEnumForClient` enum from SHC as well, this is greatly help in merges in SHE in the future.

Reply:
Sure